### PR TITLE
Qian

### PR DESCRIPTION
**Pull request by qianzhang5**

### CODE REVIEW

File: src/message.rs [hunk 1]
```diff
@@ -159,6 +159,10 @@ pub enum Method {
     Setup,
     /// Set parameter
     SetParameter,
+    /// Announce
```
Comment (sdroege) [inline src/message.rs:162]: Similar to `PlayNotify`, please document here that this is only for RTSP 1.0.

**Review by sdroege — COMMENTED** [review]
Looks good to me otherwise, thanks!

Please squash that documentation fixup into the corresponding commit instead of adding a new commit.

File: src/message.rs (hunk 1 above)
Comment (qianzhang5) [reply src/message.rs]: It's changed and done within a commit.

**Comment by qianzhang5** [comment]
Done.

File: src/message.rs [hunk 2]
```diff
@@ -159,6 +159,10 @@ pub enum Method {
     Setup,
     /// Set parameter
     SetParameter,
+    /// Announce (only RTSP 1.0)
+    Announce,
+    /// Record
+    Record,
```
Comment (sdroege) [inline src/message.rs:165]: The same also for `Record`. That's only RTSP 1.0 too.

Comment (sdroege) [reply src/message.rs:165]: Looks good otherwise, thanks!

**Comment by qianzhang5** [comment]
Done.

**Review by sdroege — APPROVED** [review]

**Pull request merged**